frontend/packages/kubevirt-plugin/src/components/vm-disks/utils.ts

```diff
@@ -14,5 +14,6 @@ export const cdTableColumnClasses = [
   classNames('col-lg-4'),
   classNames('col-lg-4'),
   classNames('col-lg-4'),
+  classNames('col-lg-4'),
```

Comment:
we have only 12 columns, so it should be 3 3 3 3

Reply:
done